core/math/vector4.h

```diff
@@ -31,10 +31,8 @@
 #ifndef VECTOR4_H
 #define VECTOR4_H
 
-#include "core/error/error_macros.h"
-#include "core/math/math_funcs.h"
-
 class String;
```

Comment:
```suggestion
#include "core/typedefs.h"

class String;
```

Reply:
The danger of doing regular builds with SCU :/